index.html

```diff
@@ -63,5 +63,143 @@ <h2 class="let">LET'S CONNECT</h2>
       
     </div>
     </section>
+    <section class="works">
+      <div class="card">
+        <div class="card__image">
+          <img src="images/tox1.png" alt="">
+        </div>
+        <div class="card__content">
+          <h2 class="card__content--heading">Tonic</h2>
+          <ul class="card__content--item1">
+            <li class="card__item--heading"> Canopy</li>
+            <li> <img src="images/Counter.png" alt="Counter"> Back End Dev</li>
+            <li> <img src="images/Counter.png" alt="Counter"> 2015</li>
+          </ul>
+          <p class="card__content--desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="card__content--item2">
+            <li class="tech__btn">html</li>
+            <li class="tech__btn">css</li>
+            <li class="tech__btn">Javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      </div>
+      
+      <div class="card">
+        <div class="card__image">
+          <img src="images/mul2.png" alt="mul2.png">
+        </div>
+        <div class="card__content">
+          <h2 class="card__content--heading">Multi-Post Stories</h2>
+          <ul class="card__content--item1">
+            <li class="card__item--heading"> Canopy</li>
+            <li> <img src="images/Counter.png" alt="Counter"> Back End Dev</li>
+            <li> <img src="images/Counter.png" alt="Counter"> 2015</li>
+          </ul>
+          <p class="card__content--desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="card__content--item2">
+            <li class="tech__btn">html</li>
+            <li class="tech__btn">css</li>
+            <li class="tech__btn">Javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card__image">
+          <img src="images/ton21.png" alt="ton21">
+        </div>
+        <div class="card__content">
+          <h2 class="card__content--heading">Tonic</h2>
+          <ul class="card__content--item1">
+            <li class="card__item--heading"> Canopy</li>
+            <li> <img src="images/Counter.png" alt="Counter"> Back End Dev</li>
+            <li> <img src="images/Counter.png" alt="Counter"> 2015</li>
+          </ul>
+          <p class="card__content--desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="card__content--item2">
+            <li class="tech__btn">html</li>
+            <li class="tech__btn">css</li>
+            <li class="tech__btn">Javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
```

Comment:
- [ ] You really did well on adding the hover effects to your buttons 🚀, but you still need to match the hover effect color as in your design template. Please take a look at the comparison and try to match the design color 👇🏽

### The hover effect color you implemented
![image](https://user-images.githubusercontent.com/43172164/206498351-da3cf96a-ec37-4b4c-8c94-1094fd3a1def.png)

### The Figma design hover effect color you need to match
![image](https://user-images.githubusercontent.com/43172164/206498711-3c3af01a-31ad-4fcf-81fc-43bdabcd0f81.png)

**_Please refer to the_** [design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)